spring-kafka/src/main/java/org/springframework/kafka/listener/adapter/RecordFilterStrategy.java

```diff
@@ -58,4 +60,14 @@ default List<ConsumerRecord<K, V>> filterBatch(List<ConsumerRecord<K, V>> record
 		return records;
 	}
 
+	/**
+	 * Returns a boolean value that determines whether {@link FilteringBatchMessageListenerAdapter} invoke the {@link BatchMessageListener} when all {@link ConsumerRecord}
+	 * have been filtered and return an EmptyList. The default is not to invoke the {@link BatchMessageListener} (false).
```

Comment:
What is `EmptyList`?
This is not a word, I believe better to say `empty list`.
I mean this is an official documentation of Spring: it has to be as professional as possible.
I also believe that default `false` means to call delegate with an empty list.

Reply:
> I also believe that default false means to call delegate with an empty list.
Sorry, i made a mistake 😓😓😓.. I fixed it! 

DITTO, `EmptyList`!